routes.yaml

```diff
@@ -3,4 +3,4 @@ group: api
 
 weight: 90
 routes:
-  1: ^/auth-code-notification/*
\ No newline at end of file
+  1: ^/company/(.*)/auth-code/.*
```

Comment:
Do we need `.*` at the end or will just `*` be sufficient?

From OE API we have this for the transactions end-points:

`^/transactions/(.*)/overseas-entity/*`

Reply:
they look to be the same so I will change to *